packages/react-scripts/config/jest/cssTransform.js

```diff
@@ -13,7 +13,9 @@
 
 module.exports = {
   process() {
-    return 'module.exports = {};';
+    return {
+      code: 'module.exports = {};',
+    };
   },
```

Comment:
Returning string isn't allowed anymore with v28

https://jestjs.io/docs/upgrading-to-jest28#transformer